src/heap.py

```diff
@@ -3,6 +3,8 @@
 
 
 class Biheap:
+    """Instantiate a binary heap and its methods."""
+
     def __init__(self, iter=None):
```

Comment:
You should raise the appropriate exception for if the binheap is intialized without an iterable.